### ISSUE TITLE
Add support for countdown of world buffs that are about to pop

### ISSUE THREAD

**Issue by a17n** [BigWigsMods/BigWigs#864]
## Describe the problem

Currently BigWigs does not support countdown of world buffs (Onyxia/Nefarian, Hakkar, Rend) that are about to pop. Since the meta in Classic is to bring those to raids and the process to get them is really painful it helps a lot if you can time those on your current character or log other characters when those are about to pop.

DBM currently supports this.

### What steps will reproduce the problem?

1.  
2.  
3.

**Comment by nebularg** [comment]
Why not just use something specialized for this like https://www.curseforge.com/wow/addons/nova-world-buffs ? Persistent cooldown tracking isn't really something BigWigs handles.

**Comment by a17n** [comment]
@nebularg Nova tracks the CDs of world buffs, that is a different use case. What I am requesting is the notification and timer between the quest is turned in/the NPC yells and when the actual buff is applied to players. For example Onyxia/Nef pops 15 seconds after the quest is turned in. This is enough time for you to login your raidlogged player and receive the buff.

**Comment by nebularg** [comment]
Ah, it doesn't do zone yell callouts? That's a bit more narrow in scope and seems doable

**Comment by a17n** [comment]
@nebularg exactly. So 1) Announce the yell so you know it was handed in and will pop soon, 2) Add a countdown timer from that yell to when the actual buff is handed out.
